test/test_copyright.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The package is under BSD but this license here is Apache

Reply:
Good point.  Since this file is copied from another place (I think I copied this from https://github.com/ros2/ros2cli/blob/rolling/ros2action/test/test_copyright.py), should we change the license on this file to be BSD, or should we add in Apache-2.0 as one of the licenses on this package?

Reply:
It would be much cleaner to keep the existing license and not cause dual licensing woes. In the case the file is bordering on de minimus. As it's 7 total lines of code, two are imports, three are matching a defined prototype with annotations thus no possible expressiveness. And that leaves two lines with one function call, saving it's return value and asserting. Which has very little expressiveness. For good measure you could re-implement those 2 lines and then it being a new composition, apply the project's license.